charts/app/templates/appealsServiceApi.yaml

```diff
@@ -106,6 +108,8 @@ spec:
             - name: SRV_NOTIFY_TEMPLATE_ID
               value: {{ .Values.appealsServiceApi.config.notify.templateId | quote }}
             - name: SRV_NOTIFY_APPEAL_SUBMISSION_RECEIVED_NOTIFICATION_EMAIL_TO_LPA_TEMPLATE_ID
+              value: {{ .Values.appealsServiceApi.config.notify.templates.appealSubmissionConfirmationEmailToAppellant | quote }}
+            - name: SRV_NOTIFY_APPEAL_SUBMISSION_CONFIRMATION_EMAIL_TO_APPELLANT_TEMPLATE_ID
               value: {{ .Values.appealsServiceApi.config.notify.templates.appealNotificationEmailToLpa | quote }}
           volumeMounts:
```

Comment:
These are mixed up.
Should be:
- name: NAME_VAR_1
   value: VALUE_VAR_1

Reply:
👍